.kokoro/trampoline.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2017 Google Inc.
```

Comment:
```suggestion
# Copyright 2023 Google Inc.
```

Reply:
This will be fixed at the source in https://github.com/googleapis/synthtool/pull/1815